src/web/components/ClientSideTokenGeneration/CstgAddDomainDialog.spec.tsx

```diff
@@ -61,21 +61,12 @@ describe('CstgDomainAddDomainDialog', () => {
 
     await submitDialog();
 
-    await waitFor(() => {
-      expect(onAddDomainsMock).toHaveBeenCalledWith(
-        ['test.com', 'test2.com', 'test3.com', 'test4.com'],
-        expect.anything()
-      );
-    });
-
     expect(screen.queryByRole('alert')).not.toBeInTheDocument();
```

Comment:
We should probably assert on something stronger here - e.g. if the dialog is not submitted, this test will still pass.

Reply:
I can take this test out (expect alert to not be in the document) - but is the test above checking what it was called with should be sufficient?  I was looking at the tests in KeyCreationDialog.spec file for testing the creation of a successful key and it looks like the "toHaveBeenCalledWith" test is all that is used.   please let me know if there is a better/different solution, thanks!

Reply:
New ticket to be created for pattern changes around testing at a higher level